worker/src/data.ts

```diff
@@ -118,6 +118,7 @@ export interface AnalyticsDataHistory {
     supervised: number;
     unknown: number;
   };
+  versions?: Record<string, number>;
```

Comment:
Not sure why GitHub does not mark this as a conflict, but this change was added in #218, rebase and this is not needed.